Gemfile

```diff
@@ -38,12 +39,14 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
   gem 'bullet'
+  gem 'byebug'
```

Comment:
You don`t need this. Use pry-rails.

Reply:
Why? Byebug has some features that pry-rails doesn`t